templates/dart/lib/query.dart.twig

```diff
@@ -1,61 +1,97 @@
 part of {{ language.params.packageName }};
 
+/// Helper class to generate query strings.
 class Query {
-    Query._();
-    
+  Query._();
+
+  /// Filter resources where [attribute] is equal to [value].
+  /// 
+  /// [value] can be a single value or a list. If a list is used
+  /// the query will return resources where [attribute] is equal
```

Comment:
this part is suitable for many other filters, let's update all relevant

Reply:
You mean for the other sdks? You want me to add it for all of them now? Would it be quicker to get this in first and then work on the other soon after?

Reply:
I mean for other methods in this query file